student.rb

```diff
@@ -0,0 +1,12 @@
+class Student < Person
+  def initialize(_id, age, classroom, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] Once the `id` is set and initialized in the `Person` class, it would not need here again. Since `Student` inherits from `Person` class